frontend/src/components/Game/Card/Base.vue

```diff
@@ -1,39 +1,28 @@
 <script setup>
-import { ref, inject } from "vue";
-import useRomsStore from "@/stores/roms.js";
+import storeRoms from "@/stores/roms.js";
 import ActionBar from "@/components/Game/Card/ActionBar.vue";
 import Cover from "@/components/Game/Card/Cover.vue";
 
 // Props
-const props = defineProps(["rom", "index"]);
+const props = defineProps(["rom", "index", "selected"]);
 const emit = defineEmits(["selectRom"]);
-const romsStore = useRomsStore();
-const selected = ref();
+const romsStore = storeRoms();
```

Comment:
You sure we don't need `storeToRefs` here to keep this reactive? https://pinia.vuejs.org/core-concepts/#Destructuring-from-a-Store

Reply:
Yeah it works for sure, we don't need it to keep the reactivity in the way we are using it.

Reply:
Re-added and working!